frontend/src/actionCreators/staticContentActionCreator.js

```diff
@@ -11,144 +10,140 @@ import { createRequestHeader } from "@/utils/RequestHeaders";
         duration: 10,
       });
       dispatch(error(reducerTypes.GET_REGION_OPTIONS));
-      dispatch(hideLoading("modal"));
     });
 };
 
 export const fetchMineTenureTypes = () => (dispatch) => {
   dispatch(request(reducerTypes.GET_TENURE_TYPES));
-  dispatch(showLoading("modal"));
   return axios
     .get(ENVIRONMENT.apiUrl + API.MINE_TENURE_TYPES, createRequestHeader())
     .then((response) => {
       dispatch(success(reducerTypes.GET_TENURE_TYPES));
       dispatch(staticContentActions.storeTenureTypes(response.data));
-      dispatch(hideLoading("modal"));
     })
     .catch(() => {
       notification.error({
         message: String.ERROR,
         duration: 10,
       });
       dispatch(error(reducerTypes.GET_TENURE_TYPES));
-      dispatch(hideLoading("modal"));
     });
 };
 
 export const fetchMineTailingsRequiredDocuments = () => (dispatch) => {
   dispatch(request(reducerTypes.GET_MINE_TSF_REQUIRED_REPORTS));
-  dispatch(showLoading("modal"));
   return axios
     .get(ENVIRONMENT.apiUrl + API.MINE_TSF_REQUIRED_DOCUMENTS, createRequestHeader())
     .then((response) => {
       dispatch(success(reducerTypes.GET_MINE_TSF_REQUIRED_REPORTS));
       dispatch(staticContentActions.storeMineTSFRequiredDocuments(response.data));
-      dispatch(hideLoading("modal"));
     })
     .catch((err) => {
       notification.error({
         message: err.response ? err.response.data.error.message : String.ERROR,
         duration: 10,
       });
       dispatch(error(reducerTypes.GET_MINE_TSF_REQUIRED_REPORTS));
-      dispatch(hideLoading("modal"));
     });
 };
 
 export const fetchExpectedDocumentStatusOptions = () => (dispatch) => {
   dispatch(request(reducerTypes.GET_EXPECTED_DOCUMENT_STATUS));
-  dispatch(showLoading("modal"));
   return axios
     .get(`${ENVIRONMENT.apiUrl + API.EXPECTED_DOCUMENT}/status`, createRequestHeader())
     .then((response) => {
       dispatch(success(reducerTypes.GET_EXPECTED_DOCUMENT_STATUS));
       dispatch(staticContentActions.storeDocumentStatusOptions(response.data));
-      dispatch(hideLoading("modal"));
     })
     .catch((err) => {
       notification.error({
         message: err.response ? err.response.data.error.message : String.ERROR,
         duration: 10,
       });
       dispatch(error(reducerTypes.GET_EXPECTED_DOCUMENT_STATUS));
-      dispatch(hideLoading("modal"));
     });
 };
 
 export const setOptionsLoaded = () => (dispatch) => {
   dispatch(staticContentActions.loadedOptions(true));
 };
+
+export const fetchProvinceCodes = () => (dispatch) => {
+  dispatch(request(reducerTypes.GET_PROVINCE_CODES));
+  return axios
+    .get(ENVIRONMENT.apiUrl + API.PROVINCE_CODES, createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_PROVINCE_CODES));
+      dispatch(staticContentActions.storeProvinceCodes(response.data));
+    })
+    .catch((err) => {
+      notification.error({
+        message: "this is causing an error",
```

Comment:
Is this our desired error message?

Reply:
no, absolutely not. Forgot to change it, Good catch! 🤦🏼‍♀️